LOS_Driver/Motor_Channel/src/LOS_D_PWMChannel.cpp

```diff
@@ -1,17 +1,22 @@
-#include "PWMChannel.hpp"
+#include "LOS_D_PWMChannel.hpp"
 
 PWMChannel::PWMChannel(uint16_t pinNum, GPIO_TypeDef* port, TIM_HandleTypeDef* timer, 
                         uint16_t timerChannel) :    pin_num_(pin_num), 
                                                     port_(port), 
                                                     timer_(timer), 
                                                     timer_channel_(timer_channel) {
     HAL_TIM_PWM_Start(timer, timer_channel);
-}
 
-PWMChannel::set(uint8_t percent) {
     uint32_t prescalar = timer->Init.Prescalar;
-    uint32_t us = (percent * (MAX_SIGNAL - MIN_SIGNAL)) / 100 + MIN_SIGNAL;
     uint32_t period_ticks = timer->Init.Period;
```

Comment:
Looks like you are re declaring these, not assigning the value of the class member

Reply:
Also not sure what prescalar is doing...?

Reply:
I'm not sure about prescalar either, I pulled this over from the previous PWM and DSHOT drivers. Looking back at the original source code, doesn't seem to be used anywhere so I'll just remove it here.